tests/ml_commons/test_ml_commons_client.py

```diff
@@ -81,7 +81,21 @@ def test_execute():
         )
     except:  # noqa: E722
         raised = True
-    assert raised == False, "Raised Exception during execute API testing"
+    assert (
+        raised == False
+    ), "Raised Exception during execute API testing with dictionary"
+
+    raised = False
+    try:
+        input_json = '{"operation": "max", "input_data": [1.0, 2.0, 3.0]}'
```

Comment:
I just realized, we should get result from this input. In the try block can we also match value? In both cases.

Reply:
Also can we try to cover the catch condition. I'm planning to add start a initiative to start covering for the except cases. This can be good start. Pls lmk if you have any question.

Reply:
Can you elaborate more on the covering except part? What do you men by cover?

Reply:
Nevermind, ignore my last comment. Approving the PR.